src/analysis/convex/cone/proper.lean

```diff
@@ -15,10 +15,9 @@ linear programs, the results from this file can be used to prove duality theorem
 
```

Comment:
While you're editing this docstring, can you add the missing `# Top level heading` at the start?